contributions/catalog/requirements.txt

```diff
@@ -4,4 +4,7 @@ gunicorn==19.9.0
 requests==2.22.0
 requests-oauthlib==1.3.0
 gevent==20.9.0
+<<<<<<< HEAD
```

Comment:
@ywkim312 Looks like a merge error happened here.

Reply:
fixed the merge conflict and changed the merge base from develop to master